Libraries/LibWeb/HTML/HTMLInputElement.idl

```diff
@@ -11,7 +11,7 @@ interface HTMLInputElement : HTMLElement {
 
     [CEReactions, Reflect] attribute DOMString accept;
     [CEReactions, Reflect] attribute DOMString alt;
-    [CEReactions, Enumerated=Autocomplete, Reflect] attribute DOMString autocomplete;
```

Comment:
it's my understanding that for these Enumerated custom attributes we use, there is a dictionary or an enum somewhere with the allowed values. Since we're no longer using the auto-generated allowed values code from the generator, should the enum be deleted from wherever it's defined?

Reply:
The enum is still used for the `form` element

Reply:
Gotcha, don't mind this comment then :)